### PR TITLE
Generated benchmarking reports

### LINKED ISSUE #366: Add task to automatically add benchmark results to README
The easiest way is probably to generate an `md` file in `docs`.

### PR DESCRIPTION
**Pull request by CommanderTvis**
resolves #366

### CODE REVIEW

File: benchmarks/build.gradle.kts [hunk 1]
```diff
@@ -130,3 +140,96 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
 readme {
     maturity = ru.mipt.npm.gradle.Maturity.EXPERIMENTAL
 }
+
+internal val benchmarksProject = project
+
+internal val ISO_DATE_TIME: DateTimeFormatter = DateTimeFormatterBuilder().run {
```
Comment (altavir) [inline benchmarks/build.gradle.kts:146]: It is better to hide it into buildSrc

File: benchmarks/build.gradle.kts [hunk 2]
```diff
@@ -130,3 +140,96 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
...
+    appendLiteral('[')
+    parseCaseSensitive()
+    appendZoneRegionId()
+    appendLiteral(']')
+    toFormatter()
+}
+
+fun noun(number: Number, singular: String, plural: String) = if (number.toLong() == 1L) singular else plural
+
+rootProject.subprojects.forEach { p ->
```
Comment (altavir) [inline benchmarks/build.gradle.kts:174]: Also to buildSrc

**Review by altavir — CHANGES_REQUESTED** [review]

File: benchmarks/build.gradle.kts [hunk 3]
```diff
@@ -130,3 +134,5 @@ tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
 readme {
     maturity = ru.mipt.npm.gradle.Maturity.EXPERIMENTAL
 }
+
+addBenchmarkProperties()
```
Comment (altavir) [inline benchmarks/build.gradle.kts:138]: Is it possible to make it an extension of https://github.com/mipt-npm/gradle-tools/blob/670a4ca71ee0a9b714eaef63fa7582ad478fedf9/src/main/kotlin/ru/mipt/npm/gradle/KScienceReadmeExtension.kt#L17? Not critical though.

**Review by altavir — APPROVED** [review]

File: benchmarks/build.gradle.kts (hunk 3 above)
Comment (CommanderTvis) [reply benchmarks/build.gradle.kts:138]: This function highly depends on `benchmarks` `Project`

Comment (altavir) [reply benchmarks/build.gradle.kts:138]: ReadmeExtension has access to project via its `project` field.

**Pull request merged**